web/react-web/src/App.test.js

```diff
@@ -4,6 +4,6 @@ import App from './App';
 
 test('renders learn react link', () => {
   const { getByText } = render(<App />);
-  const linkElement = getByText(/Sample app/i);
+  const linkElement = getByText(/April 13 predictions: Personal communication/i);
```

Comment:
It's April 12

Reply:
Interesting, this was copy&pasted from the past. Changed.